### ISSUE TITLE
Error while applying a visual style (error HRESULT E_FAIL)

### ISSUE THREAD

**Issue by OrthodoxWindows** [nptr/msstyleEditor#105]
*Windows Version*: Windows 10 21H2

When I work on a visual style (for the moment, I only encountered this problem on a style), I suddenly get an error of this type :

![msstyleEditor_jId4mqZUXN](https://user-images.githubusercontent.com/100134023/210185110-0fa4329c-179c-4f8a-bfa3-b749400e5fa4.png)

Translated into English, the message reads:

_An HRESULT E_FAIL error was returned from a call to a COM component._

This happens suddenly after editing, while the style was working fine even after many edits. All changes are lost, I had to start over. I did not encounter any problem, until some modification (different from the problematic modification the previous time); again, the style is permanently blocked.
For the second occurrence, it's probably not a bad edit, because I've done the exact same edit in another visual style before, with no issues.

**Checklist**
- [X] My uxtheme.dll is patched
- [X] I am using the latest version of msstyleEditor
- [X] None of the other issues cover this

**Comment by nptr** [comment]
Best I can do is compare the _latest working version_ with the _suddenly not working version_ and see if something broke. Apart from that I can't do much probably.

**Comment by OrthodoxWindows** [comment]
I found the problem: I had added an indexed color image under an older version of msstyleEditor. As the style is blocked, I switched to the latest version, provided with a warning. But this warning does not apply to a bad image already embedded in the style. That's why the error was occurring even with the latest version. I converted the image to RGB, the style works again. Sorry for the unnecessary issue report, and thanks for the warning in the latest version.
